src/rgw/rgw_http_client.cc

```diff
@@ -119,6 +124,13 @@ struct rgw_http_req_data : public RefCountedObject {
     std::lock_guard l{lock};
     return ret;
   }
+  
+  void set_http_status(int _http_status) {
```

Comment:
this function is unused and can be removed

Reply:
I have updated according to your suggestion. Please review,thanks.